packages/reverse-proxy/custom-config/reverse-proxy-njs/index.js

```diff
@@ -0,0 +1,16 @@
+function validateCookieJwt(r) {
+  try {
+    if (r.uri === '/remix/') {
+      var cookie = r.headersIn.Cookie.split(';');
+      var SESSION_TOKEN = cookie.find((element) =>
+        element.includes('SESSION_TOKEN')
+      );
+      SESSION_TOKEN = SESSION_TOKEN.split('=')[1];
+      global.JWTverify(SESSION_TOKEN, process.env.SECRET_OR_KEY);
+    }
+    return 'true';
+  } catch (error) {
+    r.error('[error]: ' + error);
+    return 'false';
```

Comment:
Should the returns be string here?

Reply:
packages/reverse-proxy/custom-config/reverse-proxy.conf Line 19
In if condition I'm using string to compare